### DISCUSSION TITLE
Returning Speech Probabilities

### DISCUSSION THREAD

**Discussion by qiisziilbash** [snakers4/silero-vad#577] [Q&A] [answered]
Is there any function/way to get the speech probs directly?

**Comment by snakers4** [comment, marked as answer]
https://github.com/snakers4/silero-vad/blob/93b9782f28588769cb4599f521e8f186bff97fee/src/silero_vad/utils_vad.py#L291-L297

**Reply by snakers4** [reply]
Or

https://github.com/snakers4/silero-vad/blob/93b9782f28588769cb4599f521e8f186bff97fee/src/silero_vad/utils_vad.py#L56-L91
